### PR TITLE
[11.0][MIG] product_by_supplier

### PR DESCRIPTION
**Pull request by LoisRForgeFlow**
Migration to v11.

### CODE REVIEW

**Comment by pedrobaeza** [comment]
Please squash together adjacent "OCA Transbot.." commits using https://github.com/OCA/maintainer-tools/wiki/Merge-commits-in-pull-requests

**Comment by LoisRForgeFlow** [comment]
@pedrobaeza I was on my way to do that :sweat_smile:

**Comment by pedrobaeza** [comment]
It's better to do it before opening the PR, so that you don't stress CIs, making an unneeded build.

**Comment by LoisRForgeFlow** [comment]
isn't cancel when I push again?

**Comment by pedrobaeza** [comment]
Not on runbot. And on Travis, while you push, build time already consumed may be used by other build in OCA. Or in the worst case, save some digital cats :wink:

**Comment by LoisRForgeFlow** [comment]
Being for the cats, I'll do it next time :joy_cat:

File: product_by_supplier/__manifest__.py [hunk 1]
```diff
@@ -0,0 +1,26 @@
...
+
+{
+    'name': 'Product by Supplier info',
+    'version': '11.0.1.0.0',
+    'development_status': 'Mature',
+    'category': 'Purchase Management',
+    'summary': 'Show products grouped by suppliers',
+    'author': "Elico Corp,"
+              "Odoo Community Association (OCA)",
+    'website': 'http://www.elico-corp.com, ',
```
Comment (nikul-serpentcs) [inline product_by_supplier/__manifest__.py]: @lreficent Change website URL as per [this](https://github.com/OCA/maintainer-tools/blob/master/template/module/__manifest__.py#L11)

**Review by nikul-serpentcs — COMMENTED** [review]
Minor Change

File: product_by_supplier/__manifest__.py (hunk 1 above)
Comment (LoisRForgeFlow) [reply product_by_supplier/__manifest__.py]: Done

**Comment by LoisRForgeFlow** [comment]
@nikul-serpentcs Can you please update your review?

**Review by nikul-serpentcs — APPROVED** [review]
Code Review LGTM

**Review by pedrobaeza — APPROVED** [review]

**Comment by pedrobaeza** [comment]
@lreficent I have just seen that Odoo integrates this in core enabling an option in purchase section:

seleccion_006

so this module can be removed (although I know it's already merged).

**Comment by LoisRForgeFlow** [comment]
@pedrobaeza ouch, I see :cry: 

@mreficent could you handle this? we would need to remove this module and consider this in Openupgrade (checking that this module was in v10 and then activating this feature Pedro is pointing, similar to what we did with sale_proforma_report).

**Pull request merged**